### PR TITLE
Adding adjudication get for a specific performance

### PR DESCRIPTION
**Pull request by peiyaaao**
Resulting sample response:

```{
    "1": {
        "artistic_mark": null,
        "audio_url": null,
        "cumulative_mark": null,
        "id": 1,
        "notes": null,
        "performance_id": 1,
        "special_award": null,
        "tablet_id": null,
        "technical_mark": null
    },
    "2": {
        "artistic_mark": null,
        "audio_url": null,
        "cumulative_mark": null,
        "id": 2,
        "notes": null,
        "performance_id": 1,
        "special_award": null,
        "tablet_id": null,
        "technical_mark": null
    },
    "3": {
        "artistic_mark": null,
        "audio_url": null,
        "cumulative_mark": null,
        "id": 3,
        "notes": null,
        "performance_id": 1,
        "special_award": null,
        "tablet_id": null,
        "technical_mark": null
    }
}```

### CODE REVIEW

File: backend/routes/performance_routes.py [hunk 1]
```diff
@@ -1,8 +1,25 @@
...
 blueprint = Blueprint('performance', __name__, url_prefix='/performances')
 
 @blueprint.route('/')
 def main():
-	return 'Welcome to Dancefest Backend!'
\ No newline at end of file
+	return 'Welcome to Dancefest Backend!'
+
+@blueprint.route('/<performance_id>/adjudications', methods=['GET', 'POST'])
+def handle_adjudications(performance_id):
```
Comment (imruahmed) [inline backend/routes/performance_routes.py]: i think this is confusing, lets make each method type get its own function.

File: backend/routes/performance_routes.py [hunk 2]
```diff
@@ -1,8 +1,25 @@
...
+	if request.method == 'GET':
+		return get_adjudications(performance_id)
+
+def get_adjudications(performance_id):
+	dictAdj = {}
```
Comment (imruahmed) [inline backend/routes/performance_routes.py]: python convention is to use snake case. regardless tho, we should remove this.

File: backend/routes/performance_routes.py [hunk 3]
```diff
@@ -1,8 +1,25 @@
...
+	adjudications = Adjudication.get_by(performance_id=performance_id)
+
+	for adjudication in adjudications:
```
Comment (imruahmed) [inline backend/routes/performance_routes.py]: can replace this all with `return jsonify({adjudication.id: adjudication for adjudication in adjudications})`

**Comment by imruahmed** [comment]
pls fix merge conflicts

File: backend/routes/performance_routes.py (hunk 1 above)
Comment (peiyaaao) [reply backend/routes/performance_routes.py]: do you mean having the decorator specifically be for the 'GET' method and another decorator referring to the 'POST' method. Then with their own method for each?

**Pull request merged**